iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -3305,14 +3306,24 @@ void populateVMToEmitCPatterns(MLIRContext *context,
                                               vmAnalysisCache);
 
   // List ops
+  using listArgIndices = std::initializer_list<size_t>;
   patterns.insert<ListAllocOpConversion>(typeConverter, context,
                                          vmAnalysisCache);
   patterns.insert<ListOpConversion<IREE::VM::ListReserveOp>>(
-      typeConverter, context, "iree_vm_list_reserve", 0, true, vmAnalysisCache);
+      typeConverter, context, "iree_vm_list_reserve", listArgIndices{0}, true,
+      vmAnalysisCache);
   patterns.insert<ListOpConversion<IREE::VM::ListResizeOp>>(
-      typeConverter, context, "iree_vm_list_resize", 0, true, vmAnalysisCache);
+      typeConverter, context, "iree_vm_list_resize", listArgIndices{0}, true,
+      vmAnalysisCache);
   patterns.insert<ListOpConversion<IREE::VM::ListSizeOp>>(
-      typeConverter, context, "iree_vm_list_size", 0, false, vmAnalysisCache);
+      typeConverter, context, "iree_vm_list_size", listArgIndices{0}, false,
+      vmAnalysisCache);
+  patterns.insert<ListOpConversion<IREE::VM::ListCopyOp>>(
+      typeConverter, context, "iree_vm_list_copy", listArgIndices{0, 2}, true,
```

Comment:
Without taking a deeper look into the `VM::ListCopyOp`, I am not 100% sure if I get what is happening by passing `listArgIndices{0, 2}`. Apart from that, the EmitC related changes look good!

Reply:
Thanks =D
vm.list.copy's 1st and 3rd parameters are vm.list, so I use listArgIndices{0, 2} to indicate this information.

Reply:
Yes the test is also tested in EmitC!